### PR TITLE
Fix Multi-Language Support

### PR DESCRIPTION
**Pull request by asim-shrestha**
- Translates FRONT END components only based on the user local. If the user is French for example, they will be transferred to /fr
- Adds language selector in the settings menu

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -41,4 +41,13 @@ yarn-error.log*
...
\ No newline at end of file
+.swc
+
+# extracted language files
+/public/locales/$LOCALES
+
+# package-lock.json
+#package-lock.json
+
+# .eslintcache
```
Comment (awtkns) [inline .gitignore:52]:
```suggestion
```

**Review by awtkns — APPROVED** [review]

File: public/locales/en/translation.json [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  "Please consider sponsoring the project on GitHub.": "Please consider sponsoring the project on GitHub.",
+  "SUPPORT_NOW": "Support now",
+  "Embarking on a new goal:": "Embarking on a new goal:",
+  "Added task:": "Added task:",
+  "Thinking...": "Thinking...",
+  "Executing:": "Executing:",
+  "Help": "Help",
+  "Settings": "Settings",
+  "Sign Out": "Sign Out",
+  "Sing In": "Sing In",
```
Comment (awtkns) [inline public/locales/en/translation.json:17]: This should be sign in.... unless we really do want to go the musical route 🎵

**Review by awtkns — COMMENTED** [review]
image
We still may need to work on prompting in other languages

**Comment by asim-shrestha** [comment]
> image
> 
> We still may need to work on prompting in other languages

Yeah haven't touched prompting just to keep this small

**Pull request merged**